server/database/models/Card.js

```diff
@@ -13,7 +16,7 @@ let Card = bookshelf.Model.extend({
     return this.belongsTo(Status);
   },
   priority: function () {
-    return this.belongsTo(User);
+    return this.belongsTo(Priority);
   },
 });
```

Comment:
You should be using `module.export` with the `bookshelf.model` method to define this Data Model so that the bookshelf plugin, `registry` can use it.